src/js/components/LegislativePriorities.jsx

```diff
@@ -2,12 +2,16 @@ import Markdown from "react-markdown";
 import remarkGfm from "remark-gfm";
 
 import CMSComponent from "./CMSComponent";
+import Accordion from "react-bootstrap/Accordion";
```

Comment:
This import is unused right now, but I'd say it's OK for us to keep it since we might use it in future commits to this component.